mcnoah09/Aviv-QA-Web-Technical-Test
# ⚡️ Add various functions and tests to implement test automation for Demo Nopcommerce website

### ✨ Features:
- Add page object models for user registration, products, shopping cart, login, and checkout pages
- Add various helper utility functions
- Implement hook that takes screenshot of failed tests and stores them in a screenshot folder
- Implement CI job that triggers the test, generates and upload reports on push

### ✅ Tests:

- Add test to verify user signup and product purchase
- Add test to verify existing user login and product purchase
- Add test to verify invalid signup attempts
- Add test to verify cart functionality
